config/BardinIMDRadCorPXSec.xml

```diff
@@ -5,8 +5,24 @@
   Configuration sets for the Bardin-Dokuchaeva inverse muon 
   decay differential xsec algorithm with 1-loop radiative corrections
 
+  <!--
+  Configuration for the BardinIMDRadCorPXSec xsec algorithm.
+
+  Configurable Parameters:
+  ....................................................................................
+  Name                    Type     Optional   Comment                         Default
+  XSec-Integrator         alg      No                                         genie::XSecOnElectron/Default 
+  Electron-Velocity       alg      No                                         genie::ElectronVelocityMap/Default
+  N-Integration-Samples   alg      No         Reference PXSec (int)           genie::PXSecOnElectron/Default
+  NuE-XSecRelError        alg      No         Reference PXSec (double)        genie::PXSecOnElectron/Default
+  ....................................................................................
+  -->
+
   <param_set name="Default"> 
-     <param type="alg" name="XSec-Integrator"> genie::IMDXSec/Default </param>
+    <param type="alg"     name="XSec-Integrator"> genie::XSecOnElectron/Default </param>
+    <param type="alg"     name="Electron-Velocity"> genie::ElectronVelocityMap/Default  </param>
+    <param type="alg"     name ="N-Integration-Samples" > genie::PXSecOnElectron/Default  </param>
+    <param type="alg"     name ="NuE-XSecRelError" > genie::PXSecOnElectron/Default  </param>
```

Comment:
No these are fixed in the wrong way. These are supposed to be integer and double. The type was correct, you had to fix the value